docs/Coding_Conventions_and_Standards.md

```diff
@@ -109,6 +109,19 @@ void foo(gsl::span<const std::string> names) {
 * Use [SafeInt](https://github.com/dcleblanc/SafeInt) when calculating the size of memory to allocate to protect against overflow errors
   * `#include "core/common/safeint.h"`
   * search for `SafeInt<size_t>` in the code for examples
+* The following C++ warnings should never be disabled in onnxruntime VC++ projects(Required by [Binskim](https://github.com/microsoft/binskim/blob/d9afb65c89a621411efded74c27999281d87867e/src/BinSkim.Rules/PERules/BA2007.EnableCriticalCompilerWarnings.cs)).
```

Comment:
Is this the full list of BinSkim warnings or a subset of them? If it's the former, should we just link to the official BinSkim page to avoid repetition?

Reply:
Yes, the full list. But they don't have such a doc. I put a link to their source code for reference.